x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestUtils.java

```diff
@@ -101,7 +101,7 @@ public static Tuple<Version, Version> skipVersionRange(String testName) {
         Map<String, String> pairs = extractInstructions(testName);
         String versionRange = pairs.get("skip");
         if (versionRange != null) {
-            String[] skipVersions = versionRange.split("-");
+            String[] skipVersions = versionRange.split("-", Integer.MAX_VALUE);
```

Comment:
What's up here? I'd figured `split` defaulted to as many strings as there are `-`s

Reply:
Actually, we're probably moving away from these in favor of those capabilities anyway. Or features. I forget which one. The REST layer one. Either way, I think we're not using the version skipping much any more.

Reply:
Quite obscure, but the 2 params form of split keeps the empty strings.

`"-".split("-") == []`
`"a-".split("-") == ["a"]`
`"-".split("-", 2) == ["", ""]`
`"a-".split("-", 2) == ["a", ""]`

Changed this as it was _convenient_ for my PR (Skipping that null test), while it still looked like the expected behaviour of that function (It already handles an empty upper bound)

Reply:
wild